### PR TITLE
[instance] fix multiple instance compilation

### PR DESCRIPTION
**Pull request by wgtdkp**

### CODE REVIEW

**Comment by wgtdkp** [comment]
@bukepo tests added.

File: .github/workflows/simulation.yml [hunk 1]
```diff
@@ -62,6 +62,33 @@ jobs:
...
+        sudo apt update
+        sudo apt --no-install-recommends install -y g++-multilib
+        python3 -m pip install -r tests/scripts/thread-cert/requirements.txt
+    - name: Build
+      run: |
+        ./bootstrap
+        make -f examples/Makefile-simulation
+    - name: Run
+      run: |
+        make -f examples/Makefile-simulation check
```
Comment (bukepo) [inline .github/workflows/simulation.yml:88]: Add VERBOSE=1

File: .github/workflows/simulation.yml [hunk 2]
```diff
@@ -62,6 +62,33 @@ jobs:
         ./bootstrap
         make -f examples/Makefile-simulation distcheck
 
+  multiple-instance:
+    runs-on: ubuntu-18.04
+    env:
+      CFLAGS: -m32
+      CXXFLAGS: -m32
+      LDFLAGS: -m32
```
Comment (bukepo) [inline .github/workflows/simulation.yml:70]:
```suggestion
```

File: .github/workflows/simulation.yml [hunk 3]
```diff
@@ -62,6 +62,33 @@ jobs:
...
+      COVERAGE: 1
+      MULTIPLE_INSTANCE: 1
+      REFERENCE_DEVICE: 1
+      VIRTUAL_TIME: 1
+    steps:
+    - uses: actions/checkout@v2
+    - name: Bootstrap
+      run: |
+        sudo apt update
+        sudo apt --no-install-recommends install -y g++-multilib
```
Comment (bukepo) [inline .github/workflows/simulation.yml:80]:
```suggestion
```

**Pull request merged**